modules/ztp-preparing-the-ztp-git-repository-ver-ind.adoc

```diff
@@ -33,7 +35,9 @@ The following example describes a structure using user-provided CRs for differen
 │   │   ├── helix56-v413.yaml
 │   │   ├── kustomization.yaml <3>
 │   │   ├── ns.yaml
-│   │   └── source-crs/  <4>
+│   │   └── source-crs/ <4>
+│   │      └── reference-crs/ <5>
```

Comment:
we just need 
source-crs/
custom-crs/

we dont need to create folders under source-crs

Reply:
@rohennes sorry, your previous version was the correct one. Sorry for the confusion.